src/mon/PGMap.cc

```diff
@@ -94,6 +97,9 @@ void PGMapDigest::decode(bufferlist::const_iterator& p)
   decode(avail_space_by_rule, p);
   decode(purged_snaps, p);
   decode(osd_sum_by_class, p);
+  if (v >= 5) {
```

Comment:
v should be changed to struct_v

Reply:
@shraddhaag we are using struct_v that added and set by DECODE_START macro. when we encoding into a byte stream and send it to other daemons we need to tell them first the version and the length of the byte stream so they can decode it correctly, we are encoding the byte stream with our current version (class version) so we don't need to check it during the encode. but during decode, it means that we got it from another daemon, we want to decode the byte stream correctly so we check if it contains values of members by the version of the byte stream.